src/Microsoft.DocAsCode.MarkdigEngine.Extensions/CodeSnippet/HtmlCodeSnippetRenderer.cs

```diff
@@ -126,26 +135,28 @@ public HtmlCodeSnippetRenderer(MarkdownContext context)
         {
             var result = new Dictionary<string, List<CodeSnippetExtractor>>();
 
-            AddExtractorItems(new[] { "vb", "vbhtml" },
+            AddExtractorItems(new[] { "vb", "vbhtml", "everything" },
```

Comment:
This array here `vb, vbhtml` seems to be serving the same purpose as `LanguageAlias`?

Reply:
vb and vbhtml are just references to the `LanguageAlias` dictionary where you will find vb related aliases such as "vbnet", "vbscript", ".vb", ".bas", ".vbs", ".vba", etc.